app/imageUtils.js

```diff
@@ -85,6 +85,7 @@ async function __getImage(res, imageName, folder, webpFormat, width) {
         if(!fs.existsSync(filePath)) throw new Error(`Error: ${filePath} does not exist`)
         const imageBuffer = await fs.promises.readFile(filePath)
         res.set('Content-Type', mimeType)
+        res.set('Cache-Control', 'public, max-age=2592000')
```

Comment:
Remind me, pictures have unique urls, and the urls change when the picture is being updated, right?

Reply:
If so then the header is fine